### PR TITLE
Update model READMEs

### PR DESCRIPTION
**Pull request by Ashutosh-Adhikari**

### CODE REVIEW

File: models/bert/__main__.py [hunk 1]
```diff
@@ -63,9 +66,17 @@ def evaluate_split(model, processor, args, split='dev'):
 
     dataset_map = {
         'Reuters': ReutersProcessor,
+<<<<<<< HEAD
```
Comment (achyudh) [inline models/bert/__main__.py]: There seems to be an unresolved conflict in this file. Can you please resolve this?

File: models/xml_cnn/README.md [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
 
-The file will be saved in
+To test the model, you can use the following command.
 
 ```
-xml_cnn/saves/best_model.pt
+python -m models.xml_cnn --dataset Reuters --mode static --batch-size 32 --dynamic-pool-length 8 --trained-model models/xml_cnn/saves/Reuters/best_model.pt --seed 3435
 ```
 
+## Model Type
```
Comment (achyudh) [inline models/xml_cnn/README.md]: Typo: Model Types

File: models/reg_lstm/README.md [hunk 3]
```diff
@@ -10,6 +10,18 @@ To run the model on Reuters dataset on static, just run the following from the p
...
+model/reg_lstm/saves/Reuters/best_model.pt
+```
+
+To test the model, you can use the following command.
+
+```
+python -m models.reg_lstm --dataset Reuters --mode static --batch-size 32 --trained-model models/reg_lstm/saves/Reuters/best_model.pt --seed 3435
+```
+
 ## Model types
```
Comment (achyudh) [inline models/reg_lstm/README.md]: Typo: Model Types

**Review by achyudh — CHANGES_REQUESTED** [review]

**Comment by Ashutosh-Adhikari** [comment]
@achyudh you may further review the PR

File: models/bert/__main__.py [hunk 4]
```diff
@@ -63,9 +66,11 @@ def evaluate_split(model, processor, args, split='dev'):
 
     dataset_map = {
         'Reuters': ReutersProcessor,
-        'AAPD': AAPDProcessor,
         'IMDB': IMDBProcessor,
-        'SST-2': SST2Processor
+        'AAPD': AAPDProcessor,
+        'AGNews': AGNewsProcessor,
+        'Yelp2014': Yelp2014Processor,
+        'Sogou': SogouProcessor
```
Comment (achyudh) [inline models/bert/__main__.py:74]: @Ashutosh-Adhikari SST-2 dataset is nw missing from the dataset_map. Can you please add it back?

**Review by achyudh — CHANGES_REQUESTED** [review]

File: models/bert/__main__.py (hunk 4 above)
Comment (Ashutosh-Adhikari) [reply models/bert/__main__.py:74]: @achyudh sure.

**Review by achyudh — APPROVED** [review]

File: models/bert/__main__.py [hunk 5]
```diff
@@ -93,7 +99,7 @@ def evaluate_split(model, processor, args, split='dev'):
 
     train_examples = None
     num_train_optimization_steps = None
-    if args.do_train:
+    if args.trained_model:
```
Comment (achyudh) [inline models/bert/__main__.py]: This doesn't sound right. Did you try running this with and without a pre-trained model? processor.get_train_examples should execute even when we don't have a pre-trained model

**Review by achyudh — DISMISSED** [review]

File: models/bert/__main__.py (hunk 5 above)
Comment (Ashutosh-Adhikari) [reply models/bert/__main__.py]: Sorry I am retracting the last commit. I am not having this change in this PR

Comment (Ashutosh-Adhikari) [reply models/bert/__main__.py]: Alright can you merge it now!

**Review by daemon — APPROVED** [review]
